flake8_gl_codeclimate/__init__.py

```diff
@@ -115,7 +115,9 @@ def _violation_to_codeclimate_issue(cls, v):
             # issue, including deeper explanations and links to other resources.
             "categories": cls._guess_categories(v),
             "location": {
-                "path": v.filename,
+                # remove "./" at the beginning of the filename
+                # see: https://docs.gitlab.com/ee/ci/testing/code_quality.html#implementing-a-custom-tool
```

Comment:
Can you remove the "see:" from this comment, otherwise the URL is reported as too long.

Otherwise remove both lines. with the suggested `startswith("./")` it's fairly clear what's happening.